frontend/app/index.html

```diff
@@ -28,6 +28,8 @@
           <select class="custom-select mb-2 mr-sm-4 mb-sm-0" id="dataSourceSelect">
           </select>
 
+          <!-- <button type="button" class="btn btn-primary m-1" id="reloadButton" disabled><i class="fa fa-refresh fa-lg"></i> Reload</button> -->
```

Comment:
Can you drop this line from the PR? We don't want to put commented-out code in the master branch.